lib/src/container/store.rs

```diff
@@ -712,6 +716,7 @@ impl ImageImporter {
         // First download all layers for the base image (if necessary) - we need the SELinux policy
         // there to label all following layers.
         self.unencapsulate_base(&mut import, true).await?;
+        let des_layers = self.proxy.get_layer_info_json(&self.proxy_img).await?;
```

Comment:
This is going to introduce a hard requirement on a new skopeo; I'm not sure we can do that yet.  I see two options:

- Add a cargo feature - for precedence, look at https://github.com/containers/containers-image-proxy-rs/pull/36 and cdd20ecf1f0097c485c318d6b757b6ea074db845 - I can help with this.  
- Call this API only if we're fetching from `containers-storage:` (If we're not going to add a static feature, then we should have the proxy return e.g. an `Option<LayersInfo>` so we can *dynamically* check for support)